src/entrypoints/main/index.html

```diff
@@ -14,7 +14,8 @@
 		<meta property="og:type" content="website" />
 		<meta property="og:title" content="Felles datakatalog" />
 		<meta property="og:description" content="Felles Datakatalog er en oversikt over hvilke data de ulike offentlige virksomhetene har, hvordan de henger sammen og hva de betyr. Datakatalogen gjør det mulig for deg å søke i opplysningene hos offentlige virksomheter og bruke de til for eksempel analyseformål." />
-		<script src="https://web2.questback.com/Scripts/questback.popup-1.0.1.min.js"></script>
+		<!-- In order for you to use the pop-up script without watermarks, you need to download the script from the pop-up resources and host the script on your own infrastructure. -->
+		<script src="/resources/questback.popup-1.0.1.min.js"></script>
```

Comment:
Det kan være enklere å legge inn en import-setning i `src/entrypoints/main/index.tsx` for å slippe resten av endringer

```suggestion
```

Reply:
Da må jeg flytte det som initierer questback også? index.tsx legges jo inn etter index.html.

Reply:
Jeg får det ikke til å virke bare ved å flytte import til index.tsx.